packages/mui-material/src/styles/ThemeProviderWithVars.tsx

```diff
@@ -44,16 +44,17 @@ function Experimental_CssVarsProvider(props: any) {
   if (!warnedOnce) {
     console.warn(
       [
-        'MUI: The Experimental_CssVarsProvider component has been stabilized.',
+        'MUI: The Experimental_CssVarsProvider component has been merged to ThemeProvider.',
```

Comment:
```suggestion
        'MUI: The Experimental_CssVarsProvider component has been merged into ThemeProvider.',
```

But I like the "features has been ported" that @DiegoAndai proposed for another page, IMHO it's clearer that you should use `ThemeProvider`

Reply:
Updated.